### PR TITLE
Fix: fix rc/110.3 bugs, detail in comments

### PR DESCRIPTION
**Pull request by yanguoyu**
1. Fix styles when the desc or symbol is overlong
2. Close dialog when submitting success with migrating to create a new account
3. Reset the deposit value when the dialog's visible changes and show an error dialog when the deposit fails.
4. Ignore update the transaction that calls `get_transaction` failed.

### CODE REVIEW

**Review by devchenyan — APPROVED** [review]

**Review by Keith-CY — APPROVED** [review]

**Comment by Keith-CY** [comment]
`Clear tx` should be performed on initialization, e.g. when the deposit dialog opens, instead of the workflow end. Otherwise, the initial tx may be affected by other workflows. For example,

1. open a deposit dialog and enter a value
2. close the dialog and the tx is cleared
3. do some work and a tx is generated, but the post-treat is missing by accident
4. open a deposit dialog, the tx generated in step 3 will be the initial one

**Comment by yanguoyu** [comment]
> `Clear tx` should be performed on initialization, e.g. when the deposit dialog opens, instead of the workflow end. Otherwise, the initial tx may be affected by other workflows. For example,
> 
> 1. open a deposit dialog and enter a value
> 2. close the dialog and the tx is cleared
> 3. do some work and a tx is generated, but the post-treat is missing by accident
> 4. open a deposit dialog, the tx generated in step 3 will be the initial one
> 
> To be precise, it's a reset of initial tx

Actually, when the deposit dialog opens, the transaction will be refreshed. It shows the last deposit value because the deposit dialog was not destroyed. So the last deposit value exists.  So I reset the deposit value when the dialog visible changes at https://github.com/nervosnetwork/neuron/pull/2849/commits/045d09b43d9ee40f08037e9b39df1b9dc6d40443

**Comment by silySuper** [comment]
Asset Accounts-Receive page shows abnormal when token name is long.

截屏2023-09-26 15 26 14

**Comment by silySuper** [comment]
History page shows abnormal when token name is long.

https://github.com/nervosnetwork/neuron/assets/25971273/3f5831ff-e8f1-4f85-ae4b-f0c9ff485670

**Comment by yanguoyu** [comment]
@silySuper Fixed by https://github.com/nervosnetwork/neuron/pull/2849/commits/a97012f65fcf6430f3da6bff4e58b14ee7fd125f

**Comment by Keith-CY** [comment]
Everything works well on my computer now.

File: packages/neuron-wallet/src/block-sync-renderer/tx-status-listener.ts [hunk 1]
```diff
@@ -42,18 +49,28 @@ const trackingStatus = async () => {
...
+        // ignore error, get failed skip current update
       }
     })
   )
 
-  const failedTxs = txs.filter(tx => tx.status === TransactionStatus.Failed)
-  const successTxs = txs.filter(tx => tx.status === TransactionStatus.Success)
+  const failedTxs = txs.filter(
+    (tx): tx is TransactionDetail & { status: TransactionStatus.Failed } =>
+      !!tx && tx.status === TransactionStatus.Failed
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/block-sync-renderer/tx-status-listener.ts:68]:
```suggestion
      tx?.status === TransactionStatus.Failed
```

File: packages/neuron-wallet/src/block-sync-renderer/tx-status-listener.ts [hunk 2]
```diff
@@ -42,18 +49,28 @@ const trackingStatus = async () => {
...
+  )
+  const successTxs = txs.filter(
+    (tx): tx is TransactionDetail & { status: TransactionStatus.Success } =>
+      !!tx && tx.status === TransactionStatus.Success
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/block-sync-renderer/tx-status-listener.ts:72]:
```suggestion
      tx?.status === TransactionStatus.Success
```

**Review by Keith-CY — APPROVED** [review]

**Review by silySuper — APPROVED** [review]

**Pull request merged**